api/V1/Device.js

```diff
@@ -75,4 +78,40 @@ module.exports = function(app, baseUrl) {
         }
       });
   });
+
+  /**
+   * @api {get} /api/v1/devices Get list of devices
+   * @apiName GetDevices
+   * @apiGroup Devices
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiSuccess {JSON} devices List of devices.
+   *
+   * @apiUse V1ResponseError
+   */
+  app.get(
+      apiUrl,
+      passport.authenticate(['jwt'], {session: false}),
+      async (request, response) => {
+          log.info(request.method + " Request: " + request.url);
```

Comment:
I see that lines like this are everywhere. It seems like there should be some higher level handler logging these somewhere rather than each handler do this.

Not for this PR anyway...

Reply:
Yep, also need to look at a higher level handler for errors.